style.css

```diff
@@ -1,3 +1,36 @@
 #root {
-  color: red;
+  color: rgb(34, 4, 4);
+  width: 100vw;
+  height: 100vh;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
 }
+#episode-list {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+.episode {
+  width: 20%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border-radius: 18px;
+  border: solid 1px navy;
+  box-shadow: 10px 5px 10px rgb(83, 66, 54);
+  border: 5px rgb(54, 3, 61);
+  padding: 5px;
+  margin: 20px;
+}
+/* background-color: #fff;
```

Comment:
The episode images are breaking the layout when resized, you need to manage this for smaller screens